README.md

```diff
@@ -17,6 +17,11 @@ This is quite slow and can take up to one or two hours but can provide good resu
 
 ### Usage
 
+#### Install required build tools:
+
+-   This project is based upon Node.js. Therefore, there has to be a recent version of Node.js installed on your machine (version 18 or newer).
```

Comment:
The required Node.js version may be specified in the package.json. That way there is no further need to document the version in the readme.

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines

Reply:
Thanks for the hint. I have changed the package.json and readme accordingly.